src/components/@shared/ServiceCredentialVisualizer/index.module.css

```diff
@@ -9,10 +9,48 @@
   flex-direction: column;
 }
 
+.title {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  cursor: pointer;
+}
+
+.toggle {
+  composes: toggle from '../Accordion/index.module.css';
+}
+
+.toggle svg {
+  display: inline-block;
+  width: var(--font-size-base);
+  height: var(--font-size-base);
+  fill: var(--font-color-title);
+  vertical-align: middle;
+  transition: 0.2s ease-out;
+}
+
+.open .toggle svg {
+  transform: rotate(180deg);
+}
+
 .markdownContainer {
   position: relative;
 }
 
+.container .markdownContainer {
+  max-height: 0;
+  visibility: hidden;
+  overflow: hidden;
+  transition: 0.3s;
+}
+
+.open .markdownContainer {
+  max-height: 800px;
+  visibility: visible;
+  overflow-y: scroll;
+  transition: 0.3s max-height;
```

Comment:
comment: we should think about using variables for transitions for consistency

could also become its own issue to tackle this on the repo instead of just this PR

Reply:
Agreed, would probably make sense to address this in a separate issue since there will be many files involved by the changes.